debezium-server-bom/pom.xml

```diff
@@ -12,7 +12,7 @@
     <packaging>jar</packaging>
 
     <properties>
-        <version.kinesis>2.13.13</version.kinesis>
+        <version.kinesis>2.17.241</version.kinesis>
         <version.sqs>2.13.13</version.sqs>
```

Comment:
Note that the version 2.17.241 is a very old version as well, but it's the one used in Debezium core.
Moreover, the version.sqs should use the same version as the version.kinesis to avoid incompatibilities.
But we would like to get some feedback from you about how to verify these changes and cover both kinesis and sqs (ideally, using a local machine).

Reply:
When the example was developed we tested it against rela Kinesis. Today it might be possible to use localstack to give it a try. I can't confirm it will work but it would be good to check.

Reply:
Using localstack works fine, I can rebase this PR with https://github.com/debezium/debezium-server/pull/140, so we can add an additional sink job to validate this change and also cover the Kinesis sink, wdyt?

Reply:
@Sgitario Sounds good! Could you please create a Jira and attribute commits in this PR to it using DBZ-xxx prefix in commit message. We'll merge the PR then and you can add the localstackt to the server validation PR.

Reply:
Done: https://issues.redhat.com/browse/DBZ-8517